src/components/flow/ResultsAsSimulation.tsx

```diff
@@ -243,49 +247,59 @@ export function ResultsAsSimulation() {
 		let lightColor = LightColors.BLACK;
 		const carNumber = cars.get(point.index)!;
 
-		const result = [];
-
+		let result = [
+			<EEIBorderMarker
+				key={idx}
+				width={10 + 15 * usedLights.length}
+				yCord={point.yCord}
+				xCord={point.xCord - 7.5 * usedLights.length}
+			>
+				<SimulationNumbers>{carNumber}</SimulationNumbers>
+			</EEIBorderMarker>,
+		];
+		console.log(":");
+		console.log(usedLights.length % 2);
+		console.log("");
 		for (let i = 0; i < usedLights.length; i++) {
 			const light = usedLights[i];
 			idx = light.index;
 			lightColor = lights.get(usedLights[i].index)!;
 
-			if (i == 0) {
-				result.push(
-					<EEIPointMarker
-						key={idx}
-						color={lightColor}
-						yCord={point.yCord}
-						xCord={point.xCord}
-					>
-						<SimulationNumbers>{carNumber}</SimulationNumbers>
-					</EEIPointMarker>,
-				);
-			} else {
-				result.push(
-					<EEIPointMarker
-						key={idx}
-						color={lightColor}
-						yCord={point.yCord}
-						xCord={point.xCord + 15 * i}
-					></EEIPointMarker>,
-				);
-			}
-		}
+			const shift = 1 + i * 2 - Math.ceil((usedLights.length * 1.99) / 2);
```

Comment:
W momencie, w którym to nie bedzie już tmp to weź wynieś te numerki do jakichś constów wyżej, bo przybywa magic numbers i ja np. nie mam pojecia co one oznaczaja